docs/sources/guides/whats-new-in-v6-1.md

```diff
@@ -0,0 +1,55 @@
++++
+title = "What's New in Grafana v6.1"
+description = "Feature & improvement highlights for Grafana v6.1"
+keywords = ["grafana", "new", "documentation", "6.1"]
+type = "docs"
+[menu.docs]
+name = "Version 6.1"
+identifier = "v6.1"
+parent = "whatsnew"
+weight = -12
++++
+
+# What's New in Grafana v6.1
+
+## Highlights
+
+### Ad hoc Filtering for Prometheus
+
+{{< imgbox max-width="30%" img="/img/docs/v61/prometheus-ad-hoc.gif" caption="Ad-hoc filters variable for Prometheus" >}}
```

Comment:
This is added also?

Reply:
The image you mean? Yes - https://github.com/grafana/grafana.org/commit/e5ca7f9479a37342ae07245895fc1cf919c51d6d